src/components/BatteryIndicator/index.tsx

```diff
@@ -0,0 +1,107 @@
+import { useBattery } from 'hooks/battery';
+import styled from 'styled-components';
+
+const Container = styled.div`
+  width: fit-content;
+  height: 12px;
+  display: flex;
+  align-items: center;
+
+  --battery-healthy: #A5E07F;
+  --battery-warning: #D17F6B;
+`;
+
+const ChargeLevelContainer = styled.div`
+  height: 100%;
+  width: 20px;
```

Comment:
Can we increase this width just a tad to 24px to match the current implementation's battery width?